app/views/passengers/show.haml

```diff
@@ -17,6 +17,12 @@
   - else
     No
 %p
+  %b Mobility Device:
+  - if @passenger.mobility_device?
```

Comment:
Huh.  I don't know if one is more "correct" than the other.  I honestly had no idea that there were predicate methods defined for *all* attributes.